project/project_ps.go

```diff
@@ -1,13 +1,17 @@
 package project
 
-import (
-	"golang.org/x/net/context"
-)
+import "golang.org/x/net/context"
```

Comment:
Isn't this pretty standard convention? I think it might actually be `gofmt` that makes this change.